ProfileManager.js

```diff
@@ -891,6 +895,8 @@ export default class ProfileManager {
 }
 
 function _getProfileInvocationKeyZcap({profileId, zcaps}) {
+  // FIXME is this check required on private methods?
```

Comment:
No, only on the public ones (as per the issue title #25).

Reply:
done: https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/26/commits/bcb23b615c5085b0c6a4084934bd73c5ee66b735